rir/src/ir/RuntimeFeedback_inl.h

```diff
@@ -12,11 +12,11 @@ void CallFeedback::record(Code* caller, SEXP callee) {
     if (numTargets < MaxTargets) {
         int i = 0;
         for (; i < numTargets; ++i)
-            if (targets[i] == callee)
+            if (caller->getEntry(targets[i]) == callee)
```

Comment:
`caller->getExtraPoolEntry`

This is pretty nasty. Maybe we should make `getEntry` and `setEntry` private? It looks like only [Function.h](https://github.com/reactorlabs/rir/blob/8984e0ea0c77c227b3c56c3aca5b4ff1ae0ae3ee/rir/src/runtime/Function.h#L67) would break, but we could add a `setFunction` to `Code` (which already has a `function()` accessor).

Reply:
whaa, good catch :) yeah, there needs to be some safeguard. I agree